src/parser/warrior/arms/modules/Abilities.js

```diff
@@ -300,6 +299,9 @@ class Abilities extends CoreAbilities {
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         cooldown: 8,
         gcd: null,
+        castEfficiency: {
+          disabled: true,
```

Comment:
What is this supposed to achieve? No such prop exists as far as I am aware.

Reply:
it used to exist. I remember creating it .. but it must have been removed at some point.

It used to disable the efficiency %, the max casts, and the efficiency bar in the abilities section so that it would only say the number of casts. Cause the efficiency of a spell that will likely never get used in a fight or will get used maybe once or twice out of a max 50-60 isnt really helpful imo.

I used to hide that by just getting rid of the cooldown off it, but then made that prop so that you can keep the cooldown info there and still hide the efficiency info. Must have been removed at some point.

Reply:
So long as suggestion isn't true, it will not show the suggestion and thus the efficiency percentage. The `disabled` prop hid a lot of data that could still be useful, so it was removed and part of it was implemented in the suggestion boolean.